lib/rules/no-unused-services.js

```diff
@@ -170,7 +172,22 @@ module.exports = {
             return;
           }
 
-          if (
+          if (node.callee.name === TEMPLATE_TAG_PLACEHOLDER) {
```

Comment:
Good point! I'll wait for the next case to show up -- I'm sure there are things with the recast parsing that can be extracted out cleverly too, but I think it would be helpful to see some more usages of it first in order to know what common patterns people use.